lib/fastlane/actions/slather.rb

```diff
@@ -135,7 +139,8 @@ def self.available_options
                                        optional: true),
           FastlaneCore::ConfigItem.new(key: :ignore,
                                        env_name: "FL_SLATHER_IGNORE",
-                                       description: "Tell slather to ignore files matching a path",
+                                       description: "Tell slather to ignore files matching a path or any path from an array of paths",
+                                       is_string: false,
```

Comment:
sorry @mgrebenets, i just noticed this param, it doesn't appear to be getting used anywhere because we are inspecting the type, am i just missing where it is used?